README.md

```diff
@@ -23,6 +23,10 @@ This template currently works with:
 * Bundler 2.x
 * PostgreSQL
 
+*When using `--javascript vite`:*
+* Node 16.x
```

Comment:
Do you mind changing this to `Node 16 or later`? I've been using Vite with Node 18 and that works as well.

Reply:
Oh! I just realized that it used my local node version to populate `.node_version`, and that the template didn't specify 16.

Do you think it be better to not even mention Node 16?

Reply:
I think it is good to mention 16 as a reminder, because a lot of frontend tooling is dropping support for < 16.